ome/omero-figure
# polygon support

Supports adding of Polygons from OMERO to figure and editing them in Figure.

NB: creation of NEW polygons in figure is not supported in this PR.

To test:
 - Create and save Polygons AND Polylines in:
     - iviewer
     - Insight
     - ImageJ.
 - Open image in Figure
 - Add ROIs -> Load from OMERO -> Pick Polygon / Polyline
 - Once added, can edit (drag shape or corner handles)
 - Test Copy and Paste of Polygons/Polylines, between panels AND within 1 panel
 - Save figure, reload etc - check Polygons/Polylines saved OK.
 - Export to TIFF and PDF   *NB: Polyline not supported here yet

cc @pwalczysko 

![screen shot 2017-11-20 at 09 51 06](https://user-images.githubusercontent.com/900055/33012329-6ffb6068-cdd8-11e7-86c4-dd72eb7b585e.png)
